massgen/frontend/displays/textual_terminal_display.py

```
@@ -4388,6 +4397,14 @@ def on_unmount(self) -> None:
                 pass
             self._heartbeat_timer = None
             self._stop_stall_watchdog()
+            # If the user exits while copy mode is on, the terminal is left without
+            # mouse tracking. Restore it before the driver tears down.
+            if getattr(self, "_copy_mode_active", False):
+                try:
+                    self._set_terminal_mouse_capture(True)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)